config/config.go

```diff
@@ -58,20 +58,38 @@ type ManualWarpMessage struct {
 	destinationAddress      common.Address
 }
 
+type SupportedDestination struct {
```

Comment:
I think it'd be a good practice for us to start adding documentation above structs